inference-engine/src/gna_plugin/runtime/gna_float_runtime_op.cpp

```diff
@@ -79,10 +79,15 @@ void FP::ApplyDiagonalTransform(intel_dnn_component_t *component) {
             C[i * ldc + j] = bias[i];
         }
     }
-    for (uint32_t j = 0; j < n; j++) {
-        float *Bcol = B + j * component->num_rows_in;
-        float *Ccol = C + j * component->num_rows_out;
-        cblas_ssbmv1(CblasRowMajor, CblasLower, m, 0, 1.0, A, 1, Bcol, 1, 1.0, Ccol, 1);
+    for (uint32_t i = 0; i < m; i++) {
+        float *Brow = B + i * n;
+        float *Crow = C + i * ldc;
+        float *Arow = new float[n];
```

Comment:
consider to move outside this loop, and use std::vector<float>

Reply:
done